FastBank.Services/BankAccountService/BankAccountService.cs

```diff
@@ -105,9 +110,113 @@ public void WithdrawAmount(Customer customer, BankAccount customerBankAccount)
             }
         }
 
-        public void TransferAmountToFriend(Customer customer, BankAccount customerBankAccount, Customer friend, decimal amount)
+        public void TransferMoneyToFriend(Customer customer, BankAccount customerBankAccount, Dictionary<int, User> friends)
+        {
+            var inquiryMsg = "Please input friend's email to transfer money. (type \"quit\" for exit):";
+            var emailTypeToInput = "Friend email:";
+            var emailFriend = _menuService.InputEmail(inquiryMsg, emailTypeToInput);
+
+            if (emailFriend == "quit")
+                return;
+
+            var friend = friends.FirstOrDefault(f => f.Value.Email == emailFriend).Value;
```

Comment:
Here is possible to get NullReferenceException, because of FirstOrDefault(). 
What does this mean:
null.Value
Result: Exception